src/memex/models/__init__.py

```diff
@@ -29,4 +29,5 @@
     'DocumentMeta',
     'DocumentURI',
     'merge_documents',
+    'update_document_metadata',
```

Comment:
This makes it visible to flake8 that `update_document_metadata` is imported here in order to re-export it.